hazelcast/src/main/java/com/hazelcast/instance/DefaultAddressPicker.java

```diff
@@ -53,6 +53,8 @@
 
 class DefaultAddressPicker implements AddressPicker {
 
+    static final String PREFER_IPV4_STACK = "java.net.preferIPv4Stack";
```

Comment:
The class is package private, it doesn't hurt to open some internals as package private to ease the testing (used reflection before).